moroScript.jsx

```diff
@@ -115,6 +117,23 @@
         return rtnWord;
     }
 
+    //Process dict with count to sorted dict without count value
+    function sortAndRemoveCount(dict) {
+        var toRtn = JSON.parse(JSON.stringify(dict));
+        for(var i=0; i<toRtn.length; ++i) {
+            toRtn[i]["moroword"].sort(function(a, b) {
+                return parseFloat(b["count"]) - parseFloat(a["count"]);
```

Comment:
Is `parseFloat` needed or are you just being extra cautious here? Just curious mostly--since you set count to an integer above I would have assumed it would still have a numeric type.